### PR TITLE
bgp send-community fix

### PR DESCRIPTION
**Pull request by saichint**
This PR is for fixing couple of issues.
1. If send-community CLI is set to 'both', the platform nvgens 2 lines of running config (standard and extended) in the latest images. The fix is to expect the get command output as an array and the process it, so that, this will be compatible with older releases as well and there is no need to check for software versions which is a costly operation.
2. Occasionally, the 'bfd echo' cmd is failing the minitest due to cleanup issues. This command works on layer 3 interfaces and so the fix is to turn off the switchmode on the interface being tested.

Both these fixes are tested on n9k and n3k, however, the item 1 is not passing on n3k with latest image due to difference of behavior between n9k and n3k , once that is fixed, this will pass automatically.

### CODE REVIEW

File: lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml [hunk 1]
```diff
@@ -168,13 +168,13 @@ route_reflector_client:
...
 
 send_community:
+  multiple:
   auto_default: false
   default_value: 'none'
   nexus:
     get_value: '/^send-community(?: .*)?/'
     set_value: '<state> send-community <attr>'
   ios_xr:
-    multiple:
```
Comment (mikewiebe) [inline lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml:177]: Why did you remove multiple under xr?

File: lib/cisco_node_utils/bgp_neighbor_af.rb [hunk 2]
```diff
@@ -565,15 +565,16 @@ def default_route_reflector_client
...
+      return default_send_community if val.nil? || val.empty?
       platform == :nexus ? send_community_nexus(val) : send_community_xr(val)
     end
 
     # Nexus: <state> send-community [ both | extended | standard ]
     #  NOTE: 'standard' is default but does not nvgen on some platforms
     #  Returns: none, both, extended, or standard
     def send_community_nexus(val)
-      val = val.split.last
+      return 'both' if val.size > 1
```
Comment (mikewiebe) [inline lib/cisco_node_utils/bgp_neighbor_af.rb]: What are the specific contents of `val` when `size > 1`?  Does it nvgen both `extended` and `standard`?

File: lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml (hunk 1 above)
Comment (saichint) [reply lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml:177]: I did not remove it. I moved it above so that it is common for both xr and nexus.

File: lib/cisco_node_utils/bgp_neighbor_af.rb (hunk 2 above)
Comment (saichint) [reply lib/cisco_node_utils/bgp_neighbor_af.rb]: yes, when we execute the CLI with 'both', it generates 2 lines, one with standard and the 2nd with extended.

File: lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml (hunk 1 above)
Comment (mikewiebe) [reply lib/cisco_node_utils/cmd_ref/bgp_neighbor_af.yaml:177]: @saichint Sorry my mistake.

File: lib/cisco_node_utils/bgp_neighbor_af.rb (hunk 2 above)
Comment (mikewiebe) [reply lib/cisco_node_utils/bgp_neighbor_af.rb]: I think I would prefer we be more explicit here and inspect `val` for `extended` and `standard` then rather then just check the size.

Comment (saichint) [reply lib/cisco_node_utils/bgp_neighbor_af.rb]: Ok. Changed it to test the actual values. Since the order of the nvgen could change in the future without our knowledge, I added sort just in case.

File: lib/cisco_node_utils/bgp_neighbor_af.rb [hunk 3]
```diff
@@ -565,15 +565,17 @@ def default_route_reflector_client
...
+      return default_send_community if val.nil? || val.empty?
       platform == :nexus ? send_community_nexus(val) : send_community_xr(val)
     end
 
     # Nexus: <state> send-community [ both | extended | standard ]
     #  NOTE: 'standard' is default but does not nvgen on some platforms
     #  Returns: none, both, extended, or standard
     def send_community_nexus(val)
-      val = val.split.last
+      return 'both' if val.sort ==
```
Comment (mikewiebe) [inline lib/cisco_node_utils/bgp_neighbor_af.rb]: A bit more concise and no need to sort.

```
return 'both' if val.grep(/extended|standard/).size == 2
```

Comment (saichint) [reply lib/cisco_node_utils/bgp_neighbor_af.rb]: Done
